### ISSUE TITLE
Crash on 1903 (OS Build 18362.30 Education)

### ISSUE THREAD

**Issue by LatoyaWHunter** [chompie1337/SMBGhost_RCE_PoC#3]
Getting a crash when trying to exploit build 18362.30.
crash happen on the first function ( find_low_stub )

**Comment by LatoyaWHunter** [comment]
what exact windows build is working for you ?

**Comment by chompie1337** [comment]
Hi, this happens on the first function (find_low_stub) in all of your test cases? 

my test env is Windows 10 1903 18362.1

**Comment by LatoyaWHunter** [comment]
Yes, I run it few times on every VM, and the error was always in that funcion
